pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -82,6 +82,11 @@ type GraphBuilder struct {
 	// dependencyGraphBuilder
 	monitors    monitors
 	monitorLock sync.RWMutex
+	// stoppedResources is the set resources that have been stopped
```

Comment:
s/is the set resources/is the set of resources/

Reply:
"that have been stopped": stopped is hard to understand here, what does it mean to have resources stopped? should be the controllers for the resources that have been stopped, i think.

Reply:
You're right, it's pretty unclear. I have edited it to try to make more sense. To your second point yes, stopped resources are resources for whom the shared informer (and underlying controller and reflector) have been stopped.